js/app.js

```diff
@@ -199,20 +197,16 @@ var data = {
       'rgb(229, 226, 60)',
       'rgb(202, 239, 15)'
     ],
-    borderColor: [
-    ],
-    borderWidth: [
-    ]
+    borderColor: [],
+    borderWidth: []
   }]
 };
 
 function drawChart() {
   //Get the bar chart canvas
   var ctx = document.getElementById('chart').getContext('2d');
-
   //Updates allClicks & allNames arrays
   updateChartArrays();
-  
   //Create chart object
   var chart = new Chart(ctx, {
```

Comment:
Since you are never using the variable `chart` you do not need to assign the instance created by `new Chart` to a variable.